provider/views.py

```diff
@@ -255,7 +327,36 @@ def error_response(self, request, error, **kwargs):
 
         return self.render_to_response(ctx, **kwargs)
 
+    def get_implicit_response(self, request, client):
```

Comment:
I'm not too familiar with the details of this flow.  Is there a spec this is following that we can compare against?